### PR TITLE
Post on the revamped MongoDB cache store

### PR DESCRIPTION
**Pull request by antoniomacri**
Trying to document the work done here: https://github.com/infinispan/infinispan-cachestore-mongodb/pull/23

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -117,4 +117,8 @@ miador:
   name: Yusuf Karadag
   github: https://github.com/miador
   bio: Yusuf is a student developer at JBoss for the Google Summer of Code 2022 and a newbie software engineer. He has been contributing to the @Infinispan project.
-  image: /assets/images/blog/authors/miador.jpg
\ No newline at end of file
+  image: /assets/images/blog/authors/miador.jpg
+antoniomacri:
+  name: Antonio Macrì
+  github: https://github.com/antoniomacri
+  image: /assets/images/blog/authors/antoniomacri.jpg
```
Comment (tristantarrant) [inline _data/authors.yml:124]: No bio ? :stuck_out_tongue_closed_eyes:

File: _posts/2023-05-04-reviving-mongodb-cachestore.adoc [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+---
+
+= Reviving MongoDB cache store
+
+After a long period of inactivity, the MongoDB store is now back to life.
+It has been upgraded to work with Infinispan 15.
+
+== Reactive MongoDB driver
+
+Since the Infinispan cache store API is now fully reactive, also the store has been modified to use the reactive MongoDB driver.
```
Comment (tristantarrant) [inline _posts/2023-05-04-reviving-mongodb-cachestore.adoc:17]: "the store has also been modified"

File: _posts/2023-05-04-reviving-mongodb-cachestore.adoc [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+== Database formats
+
+The older MongoDB cache store worked by persisting keys and values as binary blobs.
+This should be the most efficient way, but it makes harder to inspect the documents stored in Mongo.
```
Comment (tristantarrant) [inline _posts/2023-05-04-reviving-mongodb-cachestore.adoc:22]: "it makes it harder"

**Review by tristantarrant — CHANGES_REQUESTED** [review]
Just some minor changes

File: _posts/2023-05-04-reviving-mongodb-cachestore.adoc (hunk 2 above)
Comment (antoniomacri) [reply _posts/2023-05-04-reviving-mongodb-cachestore.adoc]: Changed, thanks.

File: _posts/2023-05-04-reviving-mongodb-cachestore.adoc (hunk 3 above)
Comment (antoniomacri) [reply _posts/2023-05-04-reviving-mongodb-cachestore.adoc]: Fixed.

**Comment by karesti** [comment]
@antoniomacri please squash the commits and we can plan for publishing, what do you think @tristantarrant  @ryanemerson ?

**Comment by antoniomacri** [comment]
@karesti done, thanks!

**Review by ryanemerson — APPROVED** [review]
LGTM. Thanks for all the hard work on this @antoniomacri

**Pull request merged**